roles/upload-site/tasks/main.yml

```diff
@@ -106,7 +106,8 @@
   with_items:
     - "-y en acsf acsf_helper stanford_ssp paranoia"
     - 'ev "_paranoia_remove_risky_permissions();"'
-    - "-y vset sws_migration_version {{ migration_version }}"
+    - "-y --exact vset sws_migration_version {{ migration_version }}"
+    - "-y --exact vset stanford_ssp_role_map_source workgroup"
```

Comment:
Probably should set the cert and key paths as well.

Reply:
They're already set. Not remembering off the top of my head what in the codebase sets those.

Reply:
See https://handacenter-test.cardinalsites.stanford.edu/admin/config/stanford/stanford_ssp/role-mappings

Reply:
Those paths are wrong. I'll send in a PR.

Reply:
https://github.com/SU-SWS/ansible-playbooks/pull/49

Reply:
Where do these paths come from?

```
/usr/lib/ssl/certs/stanford-ssp.cert
/usr/lib/ssl/certs/stanford-ssp.key
```

Reply:
https://github.com/SU-SWS/stanford_ssp/blob/7.x-2.x/stanford_ssp.module#L1024

Gets the certs from the default openssl path as configured by the server environment. I set this up this way as it is the default place for SSL Certs to be found but we are tracking ours in the github repo so it is different.

Reply:
So shouldn't we be setting them in a post-settings-php hook in `acsf-cardinald7`, rather than soft-coding them in the migration script?

Reply:
I am fine with them being set during profile installation and migration. If you are concerned that users may change these I can set those two fields to their own permission grant.

Reply:
Oh, the profiles are setting them correctly on installation? OK, s/g, let's do it here.

Reply:
https://github.com/SU-SWS/Stanford-Drupal-Profile/blob/7.x-3.x/stanford.profile#L231
https://github.com/SU-SWS/stanford_install_tasks/blob/7.x-1.x/Environment/ACSF/Install/SAMLSettings.php#L13